frontend/app/src/screens/diff/diff-badge.tsx

```diff
@@ -75,3 +115,15 @@ export const BadgeUpdated = ({ className, ...props }: DiffBadgeProps) => {
     />
   );
 };
+
+export const CloseBadgeUpdated = ({ className }: CloseBadgeProps) => {
+  return (
+    <div
+      className={classNames(
+        "flex justify-center items-center absolute border-2 border-white -top-2 -right-2 bg-blue-200 text-blue-800 rounded-full",
+        className
+      )}>
+      <Icon icon="mdi:close" size={1} />
+    </div>
+  );
+};
```

Comment:
these 3 components looks duplicated. Is it possible to only have 1, then having the other one by passing colors in className?